tools/@aws-cdk/spec2cdk/lib/cdk/resource-decider.ts

```diff
@@ -184,7 +185,7 @@ export class ResourceDecider {
 
   private handleTagPropertyModern(cfnName: string, prop: Property, variant: TagVariant) {
     const originalName = propertyNameFromCloudFormation(cfnName);
-    const originalType = this.converter.makeTypeResolvable(this.converter.typeFromProperty(prop));
```

Comment:
Why this change?

Reply:
This line will the the tags type from `Array<cdk.CfnTag>` into `IResolvable | Array<cdk.CfnTags | IResolvable>`. This is not ideal as it will cause exception [here](https://github.com/aws/aws-cdk/blob/dd7efb11ba846fba2fd63db57c27cd3f210705c6/tools/@aws-cdk/spec2cdk/lib/cdk/cloudformation-mapping.ts#L187) since `originalType` doesn't find `Array<cdk.CfnTags | IResolvable>` type.

Removing the resolvable is ideal since this feature was just enabled and guaranteed to not cause any breaking changes. Fixing it at the exception location is less ideal since the code is used in other generation steps as well.